### ISSUE TITLE
[ISSUE] Fails if all workers are offline

### ISSUE THREAD

**Issue by Gibby** [ThomasPrior/EthermineInfo#15]
```Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 487, in async_update_ha_state
    await self.async_device_update()
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 691, in async_device_update
    raise exc
  File "/usr/local/lib/python3.9/concurrent/futures/thread.py", line 52, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/usr/src/homeassistant/homeassistant/util/__init__.py", line 195, in wrapper
    result = method(*args, **kwargs)
  File "/config/custom_components/ethermineinfo/sensor.py", line 190, in _update
    self._state = r.json()['data']['currentStatistics']['activeWorkers']
KeyError: 'activeWorkers'```

**Comment by ThomasPrior** [comment]
Thank you, I'll take a look at this as soon as I can.

**Comment by ThomasPrior** [comment]
I'm unable to replicate this as written.

If I spin up a miner and then turn it off the reported hashrate attribute drops to 0 and no entries are posted to the log file.

Can you please provide more detail about the scenario that creates the message you see in the log?

**Comment by ThomasPrior** [comment]
Closed in v1.0.12
